entities/global.ent

```diff
@@ -461,7 +461,7 @@
 <!ENTITY url.svn.manual.status   "http://svnbook.red-bean.com/en/1.2/svn.ref.svn.c.status.html">
 <!ENTITY url.svn.manual.update   "http://svnbook.red-bean.com/en/1.2/svn.ref.svn.c.update.html">
 <!ENTITY url.svn.manual.externaldifftools "http://svnbook.red-bean.com/en/1.2/svn.advanced.externaldifftools.html">
-<!ENTITY url.swoole.docs "https://www.swoole.co.uk/docs/">
+<!ENTITY url.swoole.docs "https://wiki.swoole.com/">
```

Comment:
Is there not an English link?

Reply:
Yes, this is a Chinese development and I noticed that they don’t really like to localize into other languages (not only in this tool)

the old link (which I am replacing) leads to some kind of fork, which is already quite different from the original swoole (as I can understand)

Reply:
Welp, I suppose people will figure it out.